pre-docs/bolt_configuration_options.md

```diff
@@ -26,6 +26,8 @@ ssh:
 
 `color`: Whether to use colored output when printing messages to the console.
 
+`hiera-config`: Set the path to your hiera config. The default is `hiera.yaml` inside the `Boltdir`.
```

Comment:
```suggestion
`hiera-config`: Specify the path to your Hiera config. The default path is `Boltdir/hiera.yaml`.
```

Reply:
that is not the default path. the default path is  something more like <Boltdir>/hiera.yaml

Reply:
The phrasing `hiera.yaml inside the Boltdir` is how we describe it for other options.